src/hotspot/share/jvmci/jvmciCompilerToVMInit.cpp

```diff
@@ -161,7 +161,8 @@ void CompilerToVM::Data::initialize(JVMCI_TRAPS) {
 }
 
 JVMCIObjectArray CompilerToVM::initialize_intrinsics(JVMCI_TRAPS) {
-  JVMCIObjectArray vmIntrinsics = JVMCIENV->new_VMIntrinsicMethod_array(vmIntrinsics::ID_LIMIT - 1, JVMCI_CHECK_NULL);
+  int len = static_cast<int>(vmIntrinsics::ID_LIMIT) - 1;
```

Comment:
vmIntrinsics::number_of_intrinsics?

Reply:
Thanks for the review. I have pushed a new version that addresses your comments.